### PR TITLE
Trending market + latest Trades section 

### PR DESCRIPTION
**Pull request by Robiquet**

### CODE REVIEW

File: components/front-page/TrendingMarketsCompact.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  });
+
+  const { data: cmsMetadata } = useMarketCmsMetadata(marketId);
+
+  const prediction = getCurrentPrediction(market.assets, market);
+
+  return (
+    <Link
+      href={`/markets/${marketId}`}
+      className="flex h-[70px] items-center p-4 hover:bg-[#D4E7F4]"
```
Comment (robhyrk) [inline components/front-page/TrendingMarketsCompact.tsx:48]: is the background colour a one-off or might it be best to add it into tailwind config?

**Review by robhyrk — CHANGES_REQUESTED** [review]
I noticed that latest trades are pulling from NTT markets. I think rather than excluding specific foreign assets in the queries filter we should filter to only include assets we use within Zeitgeist (DOT, ZTG, USDCD) in case we get more foreign assets like WSX/NTT in the future.

File: components/front-page/TrendingMarketsCompact.tsx (hunk 1 above)
Comment (Robiquet) [reply components/front-page/TrendingMarketsCompact.tsx:48]: Not sure what the plan is, I just default to adding it like this until we get duplicates

**Comment by Robiquet** [comment]
> I noticed that latest trades are pulling from NTT markets. I think rather than excluding specific foreign assets in the queries filter we should filter to only include assets we use within Zeitgeist (DOT, ZTG, USDCD) in case we get more foreign assets like WSX/NTT in the future.

Could you create a separate issue for this?

**Review by robhyrk — APPROVED** [review]

**Pull request merged**